packages/core/test/support.ts

```diff
@@ -147,15 +146,6 @@ export function createSequelizeInstance(options: Options<AbstractDialect> = {}):
     sequelizeOptions.storage = config.storage;
   }
 
-  if (dialect === 'db2') {
```

Comment:
is that intended?

Reply:
Kinda? I didn't see the other dialects doing it, so I removed it here to prevent clutter. For postgres it should still remain due to native.
I know it needs to be re-added in the future but we can make it nicer when all dialects are extracted. Let that 'some point' be before we release the first beta so users don't need to update twice